src/utils/PayloadValidation.ts

```diff
@@ -38,3 +39,9 @@ export const populateFields = (techRecord: ITechRecord) => {
   techRecord.bodyType.code = populateBodyTypeCode(techRecord.bodyType.description);
   techRecord.vehicleClass.code = populateVehicleClassCode(techRecord.vehicleClass.description);
 };
+
+export const isValidSearchCriteria = (specifiedCriteria: string): boolean => {
+  const vals: string[] = Object.values(SEARCHCRITERIA);
+  // return vals.includes(specifiedCriteria); //TODO reinstate for proper input validation
```

Comment:
Is this TODO still valid?

Reply:
Yes. This really should be switched on, but there's no scope for testing it in current stories, so opted to leave it off for now.